app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1>User profile</h1>
\ No newline at end of file
+<div>
+<%= render partial: "render", users: @users%>
+<div class="bio-container">
+<h2>Bio</h2>
+<p><%= @users[0].bio %></p>
+</div>
+
+<% @posts.each do |post| %>
+<ul class="main-container">
+    <li class="blog">
+    <a href="<%= user_post_path(post.author_id,post.id) %>" class="blog">
+    <h2><%= post.title %></h2>
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you used the `link_to` method instead of the regular anchor tag `<a>`.